demo/src/screens/componentScreens/WithScrollEnablerScreen/AutoLockScrollView.tsx

```diff
@@ -0,0 +1,89 @@
+import _ from 'lodash';
+import React, {memo} from 'react';
+import {
+  ScrollView,
+  ScrollViewProps,
+  StyleSheet,
+  LayoutChangeEvent
+} from 'react-native';
+import {
+  Colors,
+  Text,
+  View,
+  withScrollEnabler,
+  WithScrollEnablerProps
+} from 'react-native-ui-lib';
+
+export type AutoLockScrollViewProps = ScrollViewProps & {
+  numberOfItems: number;
+};
+
+const AutoLockScrollView = (props: AutoLockScrollViewProps) => {
+  const numberOfItems = props.numberOfItems;
+
+  const WithScrollEnabler = withScrollEnabler(
+    (props: WithScrollEnablerProps) => {
+      function renderItem(index: number) {
```

Comment:
You must use `useCallback` in order to define methods inside a function component. 
Otherwise what will happen is that on every render you will re-create those functions.

Reply:
Changed to useCallback